_data/it/general.yml

```diff
@@ -0,0 +1,12 @@
+footer:
+  alt:
```

Comment:
I'm really sorry ;-(. You now need to add the copyright text here.

In english it's `Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/corrados/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>`